cypher/cypher-docs/src/docs/dev/ql/administration/security/user-management-syntax.asciidoc

```diff
@@ -64,38 +56,36 @@ CREATE OR REPLACE USER name
 | Create a new user, or if a user with the same name exists, replace it.
 | <<administration-security-administration-dbms-privileges-user-management, CREATE USER>> and
 <<administration-security-administration-dbms-privileges-user-management, DROP USER>>
-| `+`
-| `+`
 
-| [source, cypher]
+| [source]
 ----
 ALTER USER name SET {
-[PLAINTEXT \| ENCRYPTED] PASSWORD password
-            [[SET PASSWORD] CHANGE [NOT] REQUIRED]
-            [SET STATUS {ACTIVE \| SUSPENDED} ] \|
+  [PLAINTEXT \| ENCRYPTED] PASSWORD password
```

Comment:
same here, you indented this line when it shouldn't
```suggestion
[PLAINTEXT \| ENCRYPTED] PASSWORD password
```

Reply:
I have no idea what the indentation or lack of it is attempting to demonstrate here

Reply:
I think the indention is for the sub parts of the line (just to not have lines that are way to long, so you have a newline and indented)
So instead of:
```
ALTER USER name SET {
[PLAINTEXT \| ENCRYPTED] PASSWORD password [[SET PASSWORD] CHANGE [NOT] REQUIRED] [SET STATUS {ACTIVE \| SUSPENDED} ] [SET DEFAULT DATABASE databaseName] \|
PASSWORD CHANGE [NOT] REQUIRED [SET STATUS {ACTIVE \| SUSPENDED} ] [SET DEFAULT DATABASE databaseName] \|
STATUS {ACTIVE \| SUSPENDED}  [SET DEFAULT DATABASE databaseName] \|
DEFAULT DATABASE databaseName \|
}
```
we have
```
ALTER USER name SET {
[PLAINTEXT \| ENCRYPTED] PASSWORD password 
    [[SET PASSWORD] CHANGE [NOT] REQUIRED] 
    [SET STATUS {ACTIVE \| SUSPENDED} ] 
    [SET DEFAULT DATABASE databaseName] \|
PASSWORD CHANGE [NOT] REQUIRED [SET STATUS {ACTIVE \| SUSPENDED} ] [SET DEFAULT DATABASE databaseName] \|
STATUS {ACTIVE \| SUSPENDED}  [SET DEFAULT DATABASE databaseName] \|
DEFAULT DATABASE databaseName \|
}
```